images/alpine/jdk11/Dockerfile.alpine.jdk11-mvn3.6-mono5-nuget5.7

```diff
@@ -2,13 +2,13 @@ ARG SNAPSHOT=""
 FROM strongboxci/alpine:jdk11-mvn3.6-mono5$SNAPSHOT
 
 # setup-nuget.sh
-ENV NUGET_VERSION=3.4.4
+ENV NUGET_VERSION=5.7.0
 ENV NUGET_PATH=/usr/local/bin/nuget-${NUGET_VERSION}.exe
 
 # S-W-I-T expects these environment variables to be set.
-ENV NUGET_V3_VERSION=$NUGET_VERSION
-ENV NUGET_V3_PATH=$NUGET_PATH
-ENV NUGET_V3_EXEC="mono ${NUGET_PATH}"
+ENV NUGET_V5_VERSION=$NUGET_VERSION
+ENV NUGET_V5_PATH=$NUGET_PATH
+ENV NUGET_V5_EXEC="mono ${NUGET_PATH}"
```

Comment:
Your change is correct, but in this particular case you should revert the `nuget3.4` file renaming and add 2 new files for `jdk8-nuget5.7 ` and `jdk11-nuget5.7`. 

Also, you would need to add `Dockerfile.alpine.jdk8-mvn3.6-mono5-nuget5.7-choco0.10` and `Dockerfile.alpine.jdk11-mvn3.6-mono5-nuget5.7-choco0.10` files which extend from the `jdk8-nuget5.7` and `jdk11-nuget5.7` images.

You also **MUST** keep the `ENV NUGET_V3_*` env vars named as they are, because this is used in the [strongbox-web-integration-tests](https://github.com/strongbox/strongbox-web-integration-tests) project. I've added a new task for this - strongbox/strongbox#1905. If you are interested in fixing that as well - comment under the issue.

Reply:
Got it. I've applied needed changes.